instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -235,11 +235,13 @@ def _get_attributes_from_request(request):
         SpanAttributes.HTTP_TARGET: request.path,
     }
 
-    if request.host:
-        attrs[SpanAttributes.HTTP_HOST] = request.host
-
     if request.remote_ip:
-        attrs[SpanAttributes.NET_PEER_IP] = request.remote_ip
+        # NET_PEER_IP is the address of the network peer
+        # HTTP_CLIENT_IP is the address of the client, which might be different
+        # if Tornado is set to trust X-Forwarded-For headers (xheaders=True)
+        attrs[SpanAttributes.HTTP_CLIENT_IP] = request.remote_ip
+        if getattr(request, "_orig_remote_ip"):
```

Comment:
Missing a default here will raise `AttributeError`.
```suggestion
        if getattr(request, "_orig_remote_ip", None):
```